### PR TITLE
feat: react root api update

### PR DESCRIPTION
**Pull request by pchalupa**
GH card ticket "link": #ADD LINK TO GITHUB TICKET OR ISSUE

### Description

Docs: https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis

### CODE REVIEW

File: apps/example/src/index.tsx [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
-);
+const container = document.getElementById('root');
+const root = createRoot(container);
+root.render(<App />);
 
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+// reportWebVitals();
```
Comment (BrantalikP) [inline apps/example/src/index.tsx:13]: same as above:)

File: apps/example/src/index.tsx [hunk 2]
```diff
@@ -1,18 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { RtcProvider } from '@torolocos/react-rtc';
+import { createRoot } from 'react-dom/client';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+// import reportWebVitals from './reportWebVitals';
```
[the remark this answers is not in the document]
Comment (pchalupa) [reply apps/example/src/index.tsx:5]: Removed

File: apps/example/src/index.tsx (hunk 1 above)
Comment (pchalupa) [reply apps/example/src/index.tsx:13]: Removed

File: apps/example/src/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pchalupa) [reply apps/example/src/index.tsx:4]: Removed

File: apps/example/src/index.tsx (hunk 1 above)
Comment (pchalupa) [reply apps/example/src/index.tsx]: Removed

**Review by BrantalikP — APPROVED** [review]

**Pull request merged**